src/App.js

```diff
@@ -21,95 +21,41 @@ const messages = {
 addLocaleData([...en, ...es]);
 
 class App extends Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
+
+    // TODO: Consider continue determining here default instance or moving all upside,
+    // forcing inject dependencies always
+    this.sessionManager =
+      (props.dependencies && props.dependencies.sessionManager) ||
+      new OnlineSessionManager(
+        new HttpDopplerMvcClient(axios, process.env.REACT_APP_API_URL),
+        60000,
```

Comment:
what means 60000?

Reply:
Hehe, the same that it meant before, but you are right, I will make it explicit.

Reply:
it's the interval time to getUserData fromdoppler right?

Reply:
Fixed!